web/src/lib/components/Footer.svelte

```diff
@@ -2,7 +2,8 @@
 
 </script>
 
-<footer class="hidden md:footer p-1.5 bg-neutral text-neutral-content ">
+<!-- large screen -->
```

Comment:
Oh, I am just now seeing the code is duplicated between large and small. This is pretty terrible. Do you think it could be refactored to be only one?